### DISCUSSION TITLE
Temp display problems

### DISCUSSION THREAD

**Discussion by ni1zzz** [mriscoc/Ender3V2S1#1195] [Q&A] [answered]
I modded my ender 3 v2 with a phaetus rapido plus hotend and it says from the first boot that my temp is at 168°C . I already tried to change the thermistor settings with `M305 P0 R4700 T1000 C0.0` for an PT1000 thermistor. 
And if I tell the printer to heat up the nozzle, the nozzle gets hoter but the displayed temp get lower.

I have no clue what could possibly be the problem.

**Comment by mriscoc** [comment]
Disable HAS_PROUI_MAXTEMP and change TEMP_SENSOR_0

**Reply by ni1zzz** [reply]
Does this work via Gcode or do i need to edit the config.h file?

**Reply by mriscoc** [reply, marked as answer]
Any thermistor change needs a new firmware build, normal firmware is compiled for the thermistor Table # 1 (T1), there are versions for T13 and T5, but there isn't for T1000

**Reply by trevthelegend6-eng** [reply]
i have a e3d revo micro that was given to me what firmware version do i need for the thermistor as i know its different to the stock creality one
